vault/quotas/quotas.go

```diff
@@ -177,6 +177,10 @@ type Response struct {
 	// Access is the handle to reach back into the quota rule that processed the
 	// quota request. This may not be set all the time.
 	Access Access
+
+	// Headers defines any optional headers that may be returned by the quota rule
+	// to clients.
+	Headers map[string]string
```

Comment:
I don't think we have the logic to append these headers to the actual API response yet. You might want to add that.

Reply:
We do -- see changes to `http/util.go`.

Reply:
Ohh, unless you mean for lease count quotas. No, we do not handle headers for lease count quotas.